features/vat-replacement/run.js

```diff
@@ -0,0 +1,239 @@
+const ctpUtils = require('../shared/ctp-utils')
+const constant = require('./constant')
+const Promise = require('bluebird')
+
+const vatConstant = constant.vatConstant
+
+async function getTaxCategories (ctpClient) {
+    const params = {
+        queryBody: `query getTaxCategories($limit: Int) {
+                        taxCategories (limit: $limit) {
+                            results {
+                                name,
+                                id,
+                                key,
+                                description, 
+                                rates {
+                                    name, 
+                                    amount, 
+                                    country, 
+                                    includedInPrice,
+                                    id
+                                },
+                                version
+                            }
+                        }
+                    }`,
+        endpoint: 'taxCategories',
+        variables: {
+            limit: 500
+        }
+    }
+
+    try {
+        let taxCategories = []
+        for await (const batch of ctpClient.fetchGraphQlBatches(params)) {
+            await Promise.map(batch, async (taxCategory) => {
+                taxCategories.push(taxCategory)
+            }, { concurrency: 4 })
+        }
+        return taxCategories
+    } catch (e) {
+        console.log(e)
+    }
+}
+
+async function replaceTaxRate(ctpClient, taxCategoryId, updateJsonObj) {
+    try {
+
+        await ctpClient.update(ctpClient.builder.taxCategories, taxCategoryId,
+            updateJsonObj.version, updateJsonObj.actions)
+    } catch (e) {
+        console.log(e)
+    }
+}
+
+async function getGermanValidTaxRateList(taxCategories, taxRateIdToTaxCategoryMap) {
```

Comment:
Add a comment to this method what is a valid German tax rate?

Reply:
https://github.com/commercetools/commercetools-cli-scripts/pull/1/commits/dd8c0b07212ecc851217d24b0932966370cfc783